Add tip about using mysql:oracle docker image for Mac M1 users (#888)
micronaut-projects/micronaut-guides: src/docs/common/snippets/common-docker-mysql-arm.adoc

> TIP: If you are using macOS on Apple Silicon – e.g. M1, M1 Pro, etc. – Docker might fail to pull a native image for `mysql:8`. In that case substitute `mysql:oracle` and make the corresponding change for the `testcontainers datasource` in `src/test/respources/application-test.yml`.

Reviewer: ```suggestion
TIP: If you are using macOS on Apple Silicon – e.g. M1, M1 Pro, etc. – Docker might fail to pull an image for `mysql:8`. In that case substitute `mysql:oracle` and make the corresponding change for the `testcontainers datasource` in `src/test/resources/application-test.yml`.
```
